components/purchase_modal/purchase.scss

```diff
@@ -0,0 +1,153 @@
+.PurchaseModal {
+    height: 100%;
+    overflow: hidden;
+
+    >div {
+        &.processing {
+            display: none;
+        }
+        overflow: hidden;
+        color: var(--center-channel-color);
+        font-size: 16px;
+        font-family: 'Open Sans';
+        padding: 77px 107px;
+        font-weight: 600;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        flex-grow: 1;
+        width: 100%;
+        height: 100%;
+
+        .footer-text {
+            font-size: 14px;
+        }
+
+        .fineprint-text {
+            margin-top: 24px;
+            font-size: 12px;
+            line-height: 16px;
+        }
+
+        .bold-text {
+            font-weight: 600;
+        }
+
+        .normal-text {
+            font-weight: normal;
+        }
+
+        .LHS {
+            width: 25%;
+
+            .title {
+                font-size: 24px;
+            }
+
+            .image {
+                padding: 32px 0px;
+            }
+        }
+
+        .central-panel {
+            width: 50%;
+        }
+
+        .full-width {
+            border-color: var(--center-channel-color-16)
```

Comment:
Same here with the use of `--center-channel-color-rgb`

Reply:
fixed